inference-engine/src/multi_device/multi_device_plugin.cpp

```diff
@@ -293,6 +293,7 @@ IExecutableNetworkInternal::Ptr MultiDeviceInferencePlugin::LoadNetworkImpl(cons
         SetExeNetworkInfo(impl,
                           executableNetworkPerDevice.begin()->second->GetInputsInfo(),
                           executableNetworkPerDevice.begin()->second->GetOutputsInfo());
+        SetExeNetworkInfo(impl, executableNetworkPerDevice.begin()->second->GetExecGraphInfo());
```

Comment:
why do we need it? I believe MULTI does not have a single execution graph because it can be different from infer request to request. So, for MULTI we can keep exec graph as simple as a single node with multiple inputs / outputs, is not it?